### PR TITLE
RT 1.7 local_bgp_test.go , RT1.5 bgp_prefix_limit_test.go, RT1.4 bgp_graceful_restart_test.go

### LINKED ISSUE #823: bgp password should not be compared
https://github.com/openconfig/featureprofiles/blob/9cb1fb01678a7d96a56ee4871556e77b0b5dbef3/feature/bgp/tests/local_bgp_test/local_bgp_test.go#L224-L236

Setting the password is ok, but the comparison logic in https://github.com/openconfig/featureprofiles/blob/9cb1fb01678a7d96a56ee4871556e77b0b5dbef3/feature/bgp/tests/local_bgp_test/local_bgp_test.go#L314 will fail as it expects an exact match.   Instead, the password should not be compared to the configuration as we expect a DUT to obfuscate the password.

### PR DESCRIPTION
**Pull request by cprabha**
Hi,

RT1.7 local_bgp_test.go, RT1.3 bgp_prefix_limit_test.go: 
We need to configure bgp peer group for any neighbor even if we are not  enabling any routing policy.
1. Added BGP Peer group config in both scripts.
2. Added network instance type to default in both scripts.
3. Resolves #823 
Please review and approve.

RT1.4 bgp_graceful_restart_test.go :  
If graceful restart is configured, LLGR is enabled by default.  
Hence DUT keeps routing information until LLGR is expired. 
We need to add wait time to make sure LLGR is expired to test forwarding capability after Graceful restart.

Thanks,
Prabha

### CODE REVIEW

**Comment by cprabha** [comment]
Reopened after resolving conflicts.

**Comment by cprabha** [comment]
@amrindrr , Please could you review and provide approval. Thank you.

**Comment by cprabha** [comment]
Hi,

I am getting error in validating protobufs. 
Error:
====
Run arduino/setup-protoc@v1
Error: API rate limit exceeded for 104.210.132.1[7](https://github.com/openconfig/featureprofiles/actions/runs/3745148675/jobs/6359266300#step:5:8)7. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.

Please could you let me know how to fix it ? 

Thanks,
Prabha

File: feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go [hunk 1]
```diff
@@ -392,6 +392,8 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		t.Log("Configure BGP with Graceful Restart option under Global Bgp")
 		dutConfPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
 		gnmi.Delete(t, dut, dutConfPath.Config())
+		t.Log("Clear pre existing acl config on router")
+		gnmi.Delete(t, dut, gnmi.OC().Acl().Config())
```
[the remark this answers is not in the document]
Comment (cprabha) [reply feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go:396]: @ming2004 ,   if DUT config is reset then we don't need these two lines. 
I will remove them and commit the code.

**Comment by singh-prem** [comment]
could you split this PR for individual testcases.

**Comment by cprabha** [comment]
> could you split this PR for each test separately.

@singh-prem we can see the changes per file and other pull requests are also done with multiple files.
May i know if there is any specific reason for this request?

**Review by sachendras — APPROVED** [review]

**Comment by singh-prem** [comment]
> > could you split this PR for each test separately.
> 
> @singh-prem we can see the changes per file and other pull requests are also done with multiple files. May i know if there is any specific reason for this request?

Oh its just for simplicity. We need to verify all these changes don't break the tests on our platform.  With larger content

> > could you split this PR for each test separately.
> 
> @singh-prem we can see the changes per file and other pull requests are also done with multiple files. May i know if there is any specific reason for this request?

oh its just for simplicity.

**Pull request merged**